Geolocation and lock for android mobile device

mpedy opened the pull request:
Added lock option for mobile device and a geolocalization tab for getting position of android mobile device with a fork of MeshCentralAndroidAgent

CLIENT SIDE: https://github.com/Ylianst/MeshCentralAndroidAgent/pull/42

mpedy committed:
Geolocation and lock for android mobile device

si458 commented:
Unfortunately, we cannot accept this PR because the package.json has been modified with packages that won't work on Linux machines/docker

Also u say this works with a fork of the meshagent for Android rather than the official build

What might be better is u convert this into a plugin!
https://ylianst.github.io/MeshCentral/meshcentral/plugins/
Then people can install it if they want to use it

mpedy committed:
Restoring package.json and package-lock.json

mpedy commented:
I restored package.json and package-lock.json as they were previously. For Android, I'm going to check it asap thank you!

mpedy committed:
Applied changes to new UI

mpedy commented:
I think there is a typo in default3.handlebars at row 9343: `setModalContent('xxAddAgent', "Device Action", x)` should be `setModalContent('xxAddAgent', "Device Action", xxdialogTag)`. This causes the time div of the actions to not be displayed in the dialog mode

mpedy committed:
Clean map status and added custom id

mpedy committed:
Added leaflet style and js tag

Restored leaflet style tag and js tag inside generateCustomJSTags and generateCustomCSSTags

mpedy committed:
Merge remote-tracking branch 'origin/master'

mpedy committed:
Porting to OpenLayer map

mpedy committed:
Adding leaflet tag only when necessary

mpedy committed:
Completed porting to OpenLayer

mpedy committed:
Merge remote-tracking branch 'origin/master'

pofa2007 commented:
@si458 we found on old Android releases the current agent still works, on the last Android release it does not .
is there an estimated date to an apk update ?

thanks and regards
